README.md

```diff
@@ -12,6 +12,89 @@ To run docker as a non-root user, run: "sudo chmod 666 /var/run/docker.sock" (re
 To generate a new SSH key and it to the machine's SSH agent: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent?platform=linux
 Add a new SSH key to your GitHub account: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account?platform=linux
 
+## Running on a Slurm cluster
+This script must be run on a slurm cluster connected to the NFS. It will spawn all simpoints as slurm jobs to run on any node where the docker container is found. If no nodes are found it will attempt launch a container on one of the nodes using this Scarab-Infra repository (which should be on the nfs to launch containers from).
+
+### Requirements
```

Comment:
Could you document the steps to set up Slurm on the nodes for the future use if we re-install OS or so?